README.md

```diff
@@ -33,9 +33,10 @@ If you wish you can run this command under a virtual environment.
 
 ## Features
 
- * Parallel execution: If your tests independent you can run those in parallel,
-   when getting images, Avocado will create a snapshot to avoid conflicts. Also
-   domains created by tests have unique names to avoid collisions.
+ * Parallel execution: If your tests are independent of each other, you can run
+   those in parallel, when getting images, Avocado will create a snapshot to
```

Comment:
Since this is supposed to be a libvirt framework, it may be worth mentioning that the snapshot taken is using qemu-img directly and not libvirt - see, relying on a functional feature of the entity being tested during the test setup phase would not be a good idea. I'm still wondering what we're going to do if the upstream qemu + upstream libvirt will need to be tested together.

Reply:
I will do that in a different commit.